docs/install/Knative-custom-install.md

```diff
@@ -208,9 +208,9 @@ files from the Knative repositories:
 | [`monitoring.yaml`][1.2]†                      | Installs the [ELK stack][2], [Prometheus][2.1], [Grafana][2.2], and [Zipkin][2.3]**\***                                        | Serving component                                                 |
 | [`monitoring-logs-elasticsearch.yaml`][1.3]    | Installs only the [ELK stack][2]**\***                                                                                         | Serving component                                                 |
 | [`monitoring-metrics-prometheus.yaml`][1.4]    | Installs only [Prometheus][2.1]**\***                                                                                          | Serving component                                                 |
-| [`monitoring-tracing-jaeger.yaml`][1.5]        | Installs only [Jaeger][2.4].**\***                                                                                             | Serving component, ELK stack (monitoring-logs-elasticsearch.yaml) |
-| [`monitoring-tracing-jaeger-in-mem.yaml`][1.6] | Installs only [Jaeger in-memory][2.4]**\***                                                                                    | Serving component                                                 |
-| [`monitoring-tracing-zipkin.yaml`][1.7]        | Installs only [Zipkin][2.3].**\***                                                                                             | Serving component, ELK stack (monitoring-logs-elasticsearch.yaml) |
+| [`monitoring-tracing-jaeger.yaml`][1.5]        | Installs only [Jaeger][2.4]**\***                                                                                              | Serving component, ELK stack (monitoring-logs-elasticsearch.yaml), [Jaeger Operator][2.5] |
```

Comment:
Just mentioning the dependency here is not enough, we need to be clearer about what exactly should be installed and elaborate on why (explaining how to install jaeger-operator most likely should be here: https://github.com/knative/docs/blob/master/docs/serving/installing-logging-metrics-traces.md#jaeger). Please see my other comment for suggestions.

Reply:
So just to be clear, update the link referenced here to point to https://github.com/knative/docs/blob/master/docs/serving/installing-logging-metrics-traces.md#jaeger, and then add more details there?

In terms of the details in that referenced section - I would ideally not want to duplicate the instructions on the jaeger-operator readme, to avoid them getting out of step. So instead should I was thinking of just making it clear just follow the instructions in that "Installing the operator" section?

Reply:
> So just to be clear, update the link referenced here to point to https://github.com/knative/docs/blob/master/docs/serving/installing-logging-metrics-traces.md#jaeger, and then add more details there?

That line includes the * which points to that "observability install topic". Therefore, I think its okay to leave the link for "Jaeger Operator" to point to the [jaeger-operator install topic](https://github.com/jaegertracing/jaeger-operator#installing-the-operator)

Reply:
> In terms of the details in that referenced section - I would ideally not want to duplicate the instructions on the jaeger-operator readme, to avoid them getting out of step. So instead should I was thinking of just making it clear just follow the instructions in that "Installing the operator" section?

Yes, if its a lot of duplication, then lets not have to manage another copy. Im not familiar with any of the requirements here and dont have the bandwidth right now to test this end to end, so I only proposed to include the steps if they are really short (ie. 1. run kubctl .....? 2. do step two...done) 

Regardless of the number of steps, we do need to be clear about what we expect of our Knative users, and clarify what they need to do when they got to that other page. Do we want our knative user to perform each and every step listed in the [jaeger-operator install topic](https://github.com/jaegertracing/jaeger-operator#installing-the-operator)? If no, then which steps do they need to perform (ie. only a specific section, "create a new jaeger instance" too)?

If we only link them to the top of that page without context, then the assumption is that they should complete all the steps. Which is okay but we should say so (ie. "complete all the steps at ..... to install the `jaeger-operator`").